### ISSUE TITLE
Viewport jumps to top when floating element is inside container with `overflow-x: scroll;`

### ISSUE THREAD

**Issue by glenncoppens** [floating-ui/floating-ui#2498]
**Describe the bug**
On mobile devices, when a floating element is used inside a container with an "overflow-x: scroll"
 style applied, the content/viewport jumps a little (or 100% to the top).

**To Reproduce**
[Link to codesandbox](https://codesandbox.io/s/angry-lederberg-hpwksk)

1. Go to codesandbox and open app in separate browser tab (not in the codesandbox env)
2. Enable "Responsive Design Mode" (firefox) to emulate a mobile device (i used "Galaxy Note 20")
3. Scroll down to the last row in the table ans click the button in the outer most column to the right
4. You'll see that the window jumps, and that you need to scroll down to see the tooltip

**Expected behavior**
Don't jump.

**Screenshots**
Before click:
![afbeelding](https://github.com/floating-ui/floating-ui/assets/2539651/12ed3176-daf5-497c-b671-1821697257ed)

After click:
![afbeelding](https://github.com/floating-ui/floating-ui/assets/2539651/d6cd259d-1163-4afd-974e-f7a1fe8fb3d3)

**Context:**
 - OS: MacOS Ventura 13.5
 - Browser Firefox with "Responsive Design Mode" enabled

**Additional context**
* This happens on mobile devices too (that's how we got to know about this issue initially)
* I'm not sure if the cause is floating-ui related, or if this is a webbrowser quirk because the issue doesn't seem to occur when you try to reproduce this without "Responsive Design Mode"

**Comment by atomiks** [comment]
Try mounting/unmounting the tooltip element instead of leaving it in the DOM, and only running `autoUpdate`/`computePosition` once it's mounted, and cleaning it up when it unmounts. 

Otherwise try only calling `autoUpdate`/`computePosition` when it becomes visible, and cleaning it up when it's hidden.

**Comment by glenncoppens** [comment]
Thanks @atomiks !

I was using a premature composable, even from before https://github.com/lozinsky/vue-floating-ui was available. 
I thought `floating-ui/vue` didn't work for Vue 2.7 back then (we still need to migrate to 3), so I did not yet give it a new try. 

Anyway, I did now and was able to install the `floating-ui/vue` package quite easily, in which the issue does not occur anymore! 👍 

Thanks again for the feedback!
